src/common/containers/builds.js

```diff
@@ -4,14 +4,25 @@ import Helmet from 'react-helmet';
 
 import BuildHistory from '../components/build-history';
 import { Message } from '../components/forms';
+import Spinner from '../components/spinner';
+
 import { fetchSnap } from '../actions/snap-builds';
 
 import styles from './container.css';
 
 class Builds extends Component {
+  fetchInterval = null
 
   componentWillMount() {
     this.props.dispatch(fetchSnap(this.props.fullName));
+
+    this.fetchInterval = setInterval(() => {
+      this.props.dispatch(fetchSnap(this.props.fullName));
+    }, 5000);
```

Comment:
Is 5s OK or too often? We don't want to flood LP with requests, but on the other hand we want to keep user view up to date.